setup.py

```diff
@@ -138,4 +142,7 @@
 
         ],
       zip_safe=False,
+      dependency_links=[
+        'https://github.com/cezio/django-user-accounts/archive/252_255_mixed.zip#egg=django-user-accounts-2.0.2dev'
```

Comment:
Yes, although it would be used only when running ``python setup.py install`` or ``pip install --process-dependency-links path/to/local/geonode/repo``.